cypress/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "isolatedModules": false,
+    "sourceMap": false,
```

Comment:
Fix for Cypress after TS upgrade — `sourceMap: true` in our root `tsconfig.json` is incompatible with another option set by Cypress :shrug: